### DISCUSSION TITLE
Support for MacOS

### DISCUSSION THREAD

**Discussion by dqft** [maplibre/flutter-maplibre-gl#478] [Ideas]
Hi all,

Is there a future plan for supporting the macOS target platform in a near future?

Thanks,
Cheers!

**Comment by josxha** [comment]
Hi @dqft,
platform views on macOS need to be supported first to add support, see https://github.com/flutter/flutter/issues/41722.
Feel free to add support for macOS yourself and open a pull request if you are interested in adding it.
